Podcast/SubscriptionsViewController.swift

```diff
@@ -62,8 +62,8 @@ class SubscriptionsViewController: ViewController, UICollectionViewDelegate, UIC
     
     func setupCollectionViewFlowLayout() -> UICollectionViewFlowLayout {
         let layout = UICollectionViewFlowLayout()
-        let cellWidth: CGFloat = 160.5
-        let cellHeight: CGFloat = 210
+        let cellWidth: CGFloat = 0.428 * view.frame.width
```

Comment:
Where are these ratios from?

Reply:
on zeplin the screen size is 375 x 667 and the width and height given for the subscription cells is 160.5 x 210 so i wanted the cells to scale with that ratio